src/renderer/components/LoginScreen.tsx

```diff
@@ -58,9 +57,7 @@ function ImportBackupProgressDialog({
         return
       }
       onClose()
-      if ((await DeltaBackend.call('login.loadAccount', account)) === true) {
-        window.__changeScreen(Screens.Main)
-      }
+      window.__loadAccount(account)
```

Comment:
Can you describe what the problem was? I guess this is the bug that happend with the iphone backup?

Reply:
> I guess this is the bug that happend with the iphone backup?

@Jikstra No, the backup was done on Android.

Reply:
The change screen function skipped loading in the account variable so it was missing so the settings did not load for any session that was started by an backup import, sometimes there was even information of the last loaded account in the settings dialog. this pr changes the logic to use the right function for the task.